Dockerfile.dev

```diff
@@ -5,8 +5,9 @@ ARG UID=1000
 ARG GID=1000
 
 RUN dnf reinstall -y httpd-tools && \
+    dnf install -y epel-release && \
```

Comment:
EPEL on EL8 is not required to access `lua-posix`.  It comes from powertools repo.